### PR TITLE
Update default scope [SCI-4740] [SCI-4687][SCI-4746]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4740](https://biosistemika.atlassian.net/browse/SCI-4740)

### What was done
Add default scope and unscope where necessary

### CODE REVIEW

File: app/models/repository.rb [hunk 1]
```diff
@@ -22,7 +22,8 @@ class Repository < RepositoryBase
             length: { maximum: Constants::NAME_MAX_LENGTH }
 
   scope :accessible_by_teams, lambda { |teams|
-    left_outer_joins(:team_repositories)
+    with_archived
```
Comment (aignatov-bio) [inline app/models/repository.rb:25]: Do we need really it here? We are using it to get an active repository scope. Check:
`at_who_controller.rb`
`my_module_repositories_controller.rb`

File: app/models/activity.rb [hunk 2]
```diff
@@ -48,7 +48,8 @@ class Activity < ApplicationRecord
   }
 
   scope :repositories_joins, lambda {
-    joins("LEFT JOIN repositories ON subject_type = 'RepositoryBase' AND subject_id = repositories.id")
+    joins("LEFT JOIN repositories ON subject_type = 'RepositoryBase' AND subject_id = repositories.id " \
+            "AND repositories.archived != TRUE")
```
Comment (aignatov-bio) [inline app/models/activity.rb:52]: I think it will join only archived inventories

Comment (urbanrotnik) [reply app/models/activity.rb:52]: It's `!= TRUE`. So only active inventories

File: app/models/repository.rb (hunk 1 above)
Comment (urbanrotnik) [reply app/models/repository.rb:25]: Hmm... You are correct. Based on the name, I would say it should include archived, but it's better to explicitly call `.with_archived` where we need them.

**Pull request merged**